packages/clay-css/src/images/icons/asterisk.svg

```diff
@@ -4,6 +4,4 @@
 *
 * SPDX-License-Identifier: BSD-3-Clause
 -->
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
-	<path class="lexicon-icon-outline" d="M323.6,190l146.7-48.8L512,263.9l-149.2,47.6l93.6,125.2l-104.9,76.3l-96.1-126.4l-93.6,126.4L56.9,435.3l92.3-123.9L0,263.8l40.4-122.6L188.4,190v-159h135.3L323.6,190L323.6,190z"></path>
-</svg>
\ No newline at end of file
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path d="m8.792 6.982 1.72-.608L11 7.901l-1.748.593 1.096 1.559-1.229.95-1.126-1.574-1.097 1.573-1.23-.967 1.082-1.542L5 7.9l.473-1.526 1.735.608v-1.98h1.585v1.98Z"/></svg>
```

Comment:
@matuzalemsteles this is how it is in the Figma file. We need to ask @drakonux @marcoscv-work.

Reply:
Oh okay. I would say that this looks wrong because when the icon size is smaller we will almost not see the icon and this can probably cause visual alignment problems. But let's wait if this is expected.